src/vue/pages/sale-details/SaleOverview.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="sale-overview">
+    <template v-if="isLoaded">
+      <div class="sale-overview__asset">
+        <h3 class="sale-overview__asset-title">
+          {{ 'sale-overview.asset-title' | globalize }}
+        </h3>
+
+        <asset-details
+          class="sale-overview__asset-details"
+          :asset="asset"
+          :is-actions-shown="false"
+        />
+      </div>
+
+      <div class="sale-overview__details">
+        <h3 class="sale-overview__details-title">
+          {{ 'sale-overview.details-title' | globalize }}
+        </h3>
+
+        <div class="app__table sale-overview__table">
```

Comment:
move to separate component, such as you did with asset-details

Reply:
Extracted